tests/typings/sync/config.ts

```diff
@@ -27,7 +27,7 @@ const conf: WebdriverIO.Config = {
     },
     transformResponse: (response, requestOptions) => {
         if (requestOptions.method === 'DELETE' && response.statusCode === 200) {
-            response.body.deleted = true
+            console.log(response.body)
```

Comment:
Is this supposed to be a console.log?

Reply:
yes, as a check that the response object has access to a body property.